src/AzureClient/Magic/TargetMagic.cs

```diff
@@ -3,95 +3,112 @@
-                        This magic command allows for specifying or displaying the execution target for Q# job submission
-                        in an Azure Quantum workspace.
+                    The Azure Quantum workspace must have been previously initialized
+                    using the [`%azure.connect` magic command]({KnownUris.ReferenceForMagicCommand("azure.connect")})
+                    magic command. The specified execution target must be available in the workspace and support execution of Q# programs.
+
+                    #### Optional parameters
 
-                        The Azure Quantum workspace must have been previously initialized
-                        using the [`%azure.connect` magic command]({KnownUris.ReferenceForMagicCommand("azure.connect")})
-                        magic command. The specified execution target must be available in the workspace and support execution of Q# programs.
+                    - The target ID to set as the active execution target for Q# job submission. If not specified,
+                    the currently active execution target is displayed.
 
-                        #### Optional parameters
+                    #### Possible errors
 
-                        - The target ID to set as the active execution target for Q# job submission. If not specified,
-                        the currently active execution target is displayed.
+                    - {AzureClientError.NotConnected.ToMarkdown()}
+                    - {AzureClientError.InvalidTarget.ToMarkdown()}
+                    - {AzureClientError.NoTarget.ToMarkdown()}
 
-                        #### Possible errors
+                    #### Target capabilities
 
-                        - {AzureClientError.NotConnected.ToMarkdown()}
-                        - {AzureClientError.InvalidTarget.ToMarkdown()}
-                        - {AzureClientError.NoTarget.ToMarkdown()}
+                    When setting a target, the target capability is set to
+                    the maximum capability level supported by the given
+                    target. You can restrict target capability levels
+                    further by using
+                    `{controller.CommandDisplayName("target-capability")}`.
+                ".Dedent(),
+                Examples = new[]
+                {
+                    
+                    @"
+                        Set the current target for Q# job submission to `provider.qpu`:
+                        ```
+                        In []: %azure.target provider.qpu
+                        Out[]: Loading package Microsoft.Quantum.Providers.Provider and dependencies...
+                                Active target is now provider.qpu
+                                <detailed properties of active execution target>
+                        ```
+                    ".Dedent(),
+                    @"
+                        Clears the current target information:
```

Comment:
This one string literal starts with the word "Clears" - 3rd face, singular form.

But the string literal 
above starts with the word "Set", 
below - with the word "Display",
both are imperatives.

Consider making either all imperatives or all 3rd face, singular form, for consistency.